postman/schemas/paths/component/data-science-by-id.yaml

```diff
@@ -0,0 +1,64 @@
+# /component/data-science/{id}
```

Comment:
Why are we sometimes using the query parameter and sometimes the path?

Reply:
I remember, that it's quite common to use path parameters to reference an object of the class (in this case data-science) and query parameters to specify the environment around. 😇 
See https://stackoverflow.com/questions/30967822/when-do-i-use-path-params-vs-query-params-in-a-restful-api for example ;)